examples/kubernetes/otel-collector-config.yml

```diff
@@ -0,0 +1,74 @@
+receivers:
+  filelog:
+    include: [ /var/log/pods/*/*/*.log ]
```

Comment:
Once we figure out how we should exclude Collector's own logs and explain how to include if desirable. Maybe in a future PR.

Reply:
To exclude Otelcol logs:
`exclude: [ /var/log/pods/<namespace>_<pod_name>_*/<container_name>/*.log`
where `namespace`, `pod_name` and `container_name` has to be glob compliant

Reply:
The namespace or podname are not known but container name is known right? Can we use just container name? If yes, please add the `exclude` setting to the example.